test/README.md

```diff
@@ -108,12 +108,20 @@ tests will fail. If this happens, remove the cache directory (and make
 sure bitcoind processes are stopped as above):
 
 ```bash
-rm -rf cache
+rm -rf test/cache
 killall bitcoind
 ```
 
 ##### Test logging
 
+The test framework logs to two files: 
+ * `<test data directory>/test_framework.log` 
+ * `<test data directory>/node<number>/regtest/debug.log`. 
+ 
+The `<test data directory>` path is by default set to the temporary directory of
+your OS, and its full path is written as the very first line of the test output.
```

Comment:
It is not of the OS, but rather of the python environment. And it can be overwritten via a parameter to the test (runner)

Reply:
Well, the python tempfile module is used to achieve cross-platform support, but the current phrasing "...is by default set to the temporary directory of your OS" is still valid, IMO.   

Regarding the overwriting ability: My thinking was that adding  "by default" would suggest to the reader that there are other options (which are mentioned elsewhere). Brevity is nice but I could add more info if you think it is needed.

Reply:
Yeah, no strong opinion. Just for reference: https://docs.python.org/3.8/library/tempfile.html#tempfile.gettempdir